### PR TITLE
in Vagrant we need to cd into build directory to run utils/setup.php

### PR DESCRIPTION
**Pull request by mtmail**
It's not run during Vagrant provision, just instructions for users.

### CODE REVIEW

File: VAGRANT.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ is.
 
   ```
   # inside the virtual machine:
-  cd Nominatim
-    wget --no-verbose --output-document=data/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
-    ./utils/setup.php --osm-file data/monaco.osm.pbf --osm2pgsql-cache 1000 --all 2>&1 | tee monaco.$$.log
-    ./utils/specialphrases.php --countries > data/specialphrases_countries.sql
-    psql -d nominatim -f data/specialphrases_countries.sql
+  cd Nominatim/build-vagrant/
```
Comment (lonvia) [inline VAGRANT.md]: I've moved that directory. The vagrant scripts now have their build dir in /home/vagrant/build.

**Comment by mtmail** [comment]
@lonvia I changed the path in the pull-request.

File: VAGRANT.md [hunk 2]
```diff
@@ -45,11 +45,11 @@ is.
 
   ```
   # inside the virtual machine:
-  cd Nominatim
-    wget --no-verbose --output-document=data/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
-    ./utils/setup.php --osm-file data/monaco.osm.pbf --osm2pgsql-cache 1000 --all 2>&1 | tee monaco.$$.log
-    ./utils/specialphrases.php --countries > data/specialphrases_countries.sql
-    psql -d nominatim -f data/specialphrases_countries.sql
+  cd Nominatim/build/
```
Comment (lonvia) [inline VAGRANT.md]: Nope, still wrong. It should be `cd build`

**Comment by lonvia** [comment]
Fixed in ee7a6e8f927298295e4c119faecbee2736ffbfcb.

**Pull request closed without merging**